proposals/0426-bitwise-copyable.md

```diff
@@ -3,8 +3,8 @@
 * Proposal: [SE-0426](0426-bitwise-copyable.md)
 * Authors: [Kavon Farvardin](https://github.com/kavon), [Guillaume Lessard](https://github.com/glessard), [Nate Chandler](https://github.com/nate-chandler), [Tim Kientzle](https://github.com/tbkka)
 * Review Manager: [Tony Allevato](https://github.com/allevato)
-* Implementation: On `main` gated behind `-enable-experimental-feature BitwiseCopyable`
-* Status: **Accepted**
+* Implementation: in main branch of compiler
```

Comment:
Now that it's been accepted, I think it would be helpful to provide links to one or two key implementation PRs here. Could you add those?

Reply:
Added a link to the PR that made it a proper ungated feature.

Reply:
Thanks!